### PR TITLE
fixed UI bugs

### LINKED ISSUE #53: Fix UI Bugs
,**Describe the bug**
<ul>
<li>The Avtar icon is not centrally placed and does not looks nice when profile name is of multiple lines. Fix it.</li>

<li>When profile name is of multiple lines, the vibrator icon and Repeat Weekly icon doesn't appear. Change the position of these icons in Details fragment after discussing it with @rob729 .</li>

<li>The color of  vibrator and silent icon is completely different from rest of the color scheme, which doesn't looks good. Change color of vibrator and silent icon to accent color.</li>

</ul>

**Would you like to work on the issue?**
-  Yes

### PR DESCRIPTION
**Pull request by coder2699**
Fixes #53

### CODE REVIEW

**Comment by coder2699** [comment]
@rob729 Sorry, PR #54 started showing merge conflicts. When I tried to reset the head, so that the it gets solved. But instead that PR got closed. I will not repeat this mistake and be extra careful with the commands.

**Comment by coder2699** [comment]
Please check Details Fragment once, I made it scrollable and reduced the margins a little bit.

**Comment by coder2699** [comment]
@rob729 Shall I close this and try to reopen that?

**Comment by rob729** [comment]
> @rob729 Shall I close this and try to reopen that?

No

File: app/src/main/res/layout/fragment_details.xml [hunk 1]
```diff
@@ -4,34 +4,38 @@
...
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content">
     <RelativeLayout
             android:layout_width="match_parent"
             android:layout_height="wrap_content"
             android:gravity="center"
             android:padding="30dp">
 
+        <TextView
+                android:id="@+id/txt1"
```
Comment (rob729) [inline app/src/main/res/layout/fragment_details.xml]: change the id to profileTxt.

Comment (rob729) [reply app/src/main/res/layout/fragment_details.xml]: Since you are making a change, also change the id of the button from editFab to editButton.

Comment (coder2699) [reply app/src/main/res/layout/fragment_details.xml]: @rob729 I have changed the ids for both, please check

Comment (coder2699) [reply app/src/main/res/layout/fragment_details.xml]: @rob729 I have made changes that were asked. Please have a look

**Comment by coder2699** [comment]
@rob729 Please have a look at this PR

**Review by rob729 — APPROVED** [review]

**Comment by rob729** [comment]
give @coder2699 25 points

**Pull request merged**